darwin/importer/formats/superannotate.py

```diff
@@ -157,6 +166,10 @@ def _convert_objects(obj: Dict[str, Any], superannotate_classes: List[Dict[str,
     raise ValueError(f"Unknown label object {obj}")
 
 
+def _convert_tag(tag: str) -> Annotation:
+    return make_tag(tag)
```

Comment:
This function seems to basically act as proxy of `make_tag`, I'd say, let's use `make_tag` right away 🙂